packages/theming/src/utils/getColor.ts

```diff
@@ -135,6 +135,56 @@ const toProperty = (object: object, path: string) => {
   }
 };
 
+/* derive the property + transparency from the given rgba variable value */
+const fromRgba = (value: string) => {
+  let retVal;
+  const regex =
+    /rgba\s*\(\s*(?<property>[#\w.]+)\s*,\s*(?<alpha>[\w.]+)\s*\)/gu; /* ex. 'rgba(primaryHue.700, 600)' */
```

Comment:
Would it be worth making the space before the alpha value optional? Feels like an easy thing to  miss and spend a long time debugging. 🤔  E.g., `\s?`.

Reply:
Unless we have appetite to create a lint rule for this, which feels like an acceptable path as well.

Reply:
Isn't it already optional? Perhaps you can give me the valid string that fails. Here's a link for testing: https://regex101.com/r/td33LV/1

Reply:
Oops - I missed the asterisk. All good on my end!